non-supported features (#19)
mbrandenburger/fabric-rfcs: text/0000-fabric-private-chaincode-lite.md

In order to focus the development resources on the core components of FPC, the MVP `FPC Lite` excludes certain Fabric features, which will be added in the future.

- Multiple implementations for a single chaincode.
This feature is supported in Fabric v2 and gives organizations the freedom to implement and package their own chaincode.

> Ultimately, the channel can use different chaincode implementations as long as these expose the same observable actions (i.e., transaction responses), independently from the actually executed binaries.

Reviewer: In fabric you do really care only about ledger state (changes), transaction responses is a bit ambiguous (and as far as result is concerned, it is also not really (globally/publically) observable? The small fix would be to replace `transaction responses` by `ledger state changes`.  However, i think it would be better to talk about that what is essential that ledger changes must happen according to the same agreed upon state machine?  E.g., something along the lines of

```suggestion
It allows for different chaincode implementations as long as changes to the ledger state implement the same agreed upon state machine, i.e., the application integrity is ensured.
```

> FPC chaincodes have a stronger requirement: observable actions must match and non-observable actions must be known.
> The latter makes sure that a specific chaincode implementation can be examined for potential leaks of confidential data.
> Hence, clients can establish trust in how the chaincode executable treats their sensitive data.

Reviewer: To follow more in the thrust of my suggestion on above
```suggestion
FPC chaincodes have a stronger requirement: Not only must we be assured of the application integrity but we also require that all information flows be controlled to meet our confidentiality requirements.   As the execution during endorsement is unobservable by other organizations, they will require the assurance that any chaincode getting access to the state decryption keys, and hence sensitive information, will never leak unintended information.  Therefore, the implementation of a chaincode must allow for public examination for (lack of) potential leaks of confidential data.
Only then clients can establish trust in how the chaincode executable treats their sensitive data.
```

> For a given chaincode, the presented FPC architecture supports a single implementation working in a channel.

Reviewer: The channel reference seems a bit superfluous?
```suggestion
For a given chaincode, the FPC Lite currently supports only a single active implementation.
```

Author: No, because you still can update the chaincode.
Maybe we can refer to an "active" chaincode?

Reviewer: ah, that's what you meant. "only a single _active_ implementation." should indeed capture that better. (fixed above clickable suggestion accordingly).
